.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,7 +1,7 @@
 blank_issues_enabled: false
-"""
+
 contact_links:
   - name: SONIA AUV website
     url: https://sonia.etsmtl.ca/
     about: Visit our website and feel free to ask any questions.
-"""
+
```

Comment:
The wiki is missing

Reply:
Done in commit 05f598c010d5a67009b6d9a946abee833e9293aa